project/gh-action/infrastructure/kubernetes/service.yaml

```
@@ -0,0 +1,13 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: web-service
+spec:
+  type: NodePort 
+  ports:
+    - port: "80" 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)